swatch-metrics-hbi/src/main/java/com/redhat/swatch/hbi/events/services/FeatureFlags.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.swatch.hbi.events.services;
+
+import io.getunleash.Unleash;
+import jakarta.enterprise.context.ApplicationScoped;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@ApplicationScoped
+public class FeatureFlags {
```

Comment:
Would it be possible to move this up a little?  Like have 

```
@Slf4j
@ApplicationScoped
public abstract class FeatureFlags {
```
in...somewhere.  Move `swatch-unleash-import` to `swatch-common-unleash` and add this class there?

Then

```
public class HbiFeatureFlags extends FeatureFlags
```

What do you think?

Reply:
After chatting with @awood we decided that it wasn't worth the effort to do this since it didn't provide a whole lot at the moment.